src/TrayIcon.cpp

```diff
@@ -0,0 +1,31 @@
+#include "TrayIcon.hpp"
+
+
+TrayIcon::TrayIcon(MainWindow* window, Glib::RefPtr<Gtk::Builder> const& refBuilder)
+{
+    AppIndicator* trayIcon = 
+        app_indicator_new("whatsapp-for-linux", "whatsapp-for-linux", APP_INDICATOR_CATEGORY_COMMUNICATIONS);
```

Comment:
This icon doesn't work on my side. Maybe because I use i3wm. When I write "whatsapp" it works because it gets from the default theme.

Reply:
I wanted to use function `window->get_icon()`, but a name of the icon must be passed as a parameter. I don't understand how we can get name from the our icon. The icon "whatsapp" doesn't work for me.

Reply:
Hmmm, I see. I will get your latest changes and take a look at this.